pywbem/cim_operations.py

```diff
@@ -1127,6 +1169,20 @@ def last_reply_len(self):
         """
         return self._last_reply_len
 
+    @property
+    def conn_id(self):
+        """
+        :term:`unicode string`:
+        String that is attached to each log entry output to uniquely identify
+        the WBEMConnection responsible for that log.  This is also part of
+        the log name for the operation and xml logs so that logging could
+        be enabled differently for different wbem connections. It is created
+        when the WBEMConnection object is created and remains constant
+        thorughout the life of that connection. I is unique for each
+        connection.
+        """
+        return self._conn_id
```

Comment:
Am I right in assuming that the change of `conn_id` from public attribute to read-only property has nothing to do with the logging change? It is also not mentioned in the commit message or in the change log, despite the fact that we (rightfully) take away the ability for users to set the value.
-> Can we please move that into a separate PR?